scripts/report_merger/config.py

```diff
@@ -0,0 +1,61 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+"""Configuration handling for the Reports Merger."""
+
+import json
+import logging
+from configparser import NoSectionError, NoOptionError
+from json import JSONDecodeError
+
+
+from pydantic import BaseModel, ValidationError, Field
+
+from scripts.common.config import BaseConfig, InvalidConfigError, DIRECTORY_PATTERN
+
+
+class ReportsMergerConfig(BaseModel):
```

Comment:
nit: Should this be Report<i><b>s</b></i> plural or Report singular? (considering `report_merger` is singular)

Reply:
Also this below in report_merger.py

```
class ReportMerger:
    """Report Merger."""
```

Reply:
oh nice catch! I did a search and found a bunch more places where i made this mistake. They should be fixed now.